samples/common/auth.js

```diff
@@ -22,16 +23,26 @@ jQuery(document).ready(function() {
        
     force.login(
         function() {
-            console.log("Auth succeeded"); 
-            appStart();
+            console.log("Auth succeeded");
+
+            if (inBrowser) {
+                storeMap.setupUserStoreFromDefaultConfig(function() {
```

Comment:
When testing in browser, that's when the loading of static configs takes place.
NB: in the container, that work happens in SalesforceDroidGapActivity